components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/resources/publisher-api.yaml

```diff
@@ -2943,6 +2943,7 @@ definitions:
       - endpointConfig
       - visibility
       - type
+      - apiDefinition
```

Comment:
Don't we have to generate the rest api after this change

Reply:
The rest API was generated but there is no change